packages/switch/README.md

```diff
@@ -43,3 +43,7 @@ import { Switch } from '@swc-uxp-wrappers/switch';
 ```html
 <sp-switch></sp-switch>
 ```
+
+## Known Issues
+- Tab navigation does not work
```

Comment:
This should work, can you confirm it?

Reply:
This too is dependant on opacity support for input widget in PS which is in-progress